feat: Automatically Merge Meshes with BlendShapes (#1308)
anatawa12/AvatarOptimizer: .docs/content/docs/reference/merge-skinned-mesh/index.md

{{< /hint >}}

> It's better freeze & remove unchanging BlendShapes for body / cloth meshes to reduce BlendShape load.

Reviewer: ```suggestion
It's better freezing & removing unchanging BlendShapes for body / cloth meshes to reduce BlendShape load.
```

Author: ```suggestion
It's better to freeze & remove unchanging BlendShapes for body / cloth meshes to reduce BlendShape load.
```
のほうがしっくり来るかもです。

You can freeze & remove BlendShape using [Freeze BlendShape](../freeze-blendshape) component.
Add this component to both/either merge source SkinnedMeshRenderer and/or merged SkinnedMeshRenderer to freeze & remove BlendShapes.
Also, you can use `Automatically Freeze BlendShape` of [Trace and Optimize](../trace-and-optimize) component to get the same benefits.

> In previous versions of Avatar Optimizer, Unity 2019 recommended not merging face meshes due to merging BlendShape-heavy mesh will increase load on BlendShape much.

Reviewer: ```suggestion
In previous versions of Avatar Optimizer, we recommended not merging face meshes due to merging BlendShape-heavy mesh will increase load on BlendShape much in Unity 2019.
```

However, in Unity 2022, we no longer recommends not merging face meshes because the BlendShape load has been improved.

{{< hint info >}}

[Trace And Optimize](../trace-and-optimize) will automatically do the same process, so in most cases you do not need to use this component.

> {{< /hint >}}

Reviewer: ここのhint、Freeze BlendShapeみたいに上の方に置いていいと思う

Author: よいね。やります
